lib/components/NumberOfWinnersCard.jsx

```diff
@@ -0,0 +1,33 @@
+import { Card } from 'lib/components/Card'
+import { InputLabel } from 'lib/components/InputLabel'
+import { TextInputGroup, TextInputGroupType } from 'lib/components/TextInputGroup'
+import React from 'react'
+
+export const NumberOfWinnersCard = (props) => {
+  const { setNumberOfWinners, numberOfWinners } = props
+
+  return (
+    <Card>
+      <InputLabel
+        primary='Number of Winners'
+        description='The prize will be split between the selected number of winners. A single winner will be awarded the Grand Prize which will include the loot box.'
```

Comment:
Let's change this so it simply reads: "The prize will be split between the selected number of winners."

Lootboxes are likely to be only used by the main PoolTogether pools, instead of community created ones.